allennlp/training/callbacks/wandb.py

```diff
@@ -155,3 +160,12 @@ def on_start(
     def close(self) -> None:
         super().close()
         self.wandb.finish()  # type: ignore
+
+    def state_dict(self) -> Dict[str, Any]:
+        return {
+            "run_id": self._run_id,
+        }
```

Comment:
I love it when a plan comes together. I'm not sure we had any callbacks using state dicts when we put in support for those?

Reply:
Yeup!